packages/apollo-gateway/src/buildQueryPlan.ts

```diff
@@ -253,23 +253,11 @@ function splitSubfields(
   splitFields(context, path, fields, field => {
     const { parentType, fieldNode, fieldDef } = field;
 
-    const baseService = context.getBaseService(parentType);
+    const owningService =
+      context.getOwningService(parentType, fieldDef) || parentGroup.serviceName;
 
-    if (!baseService) {
```

Comment:
We don't want to remove these errors as they could still happen in unexpected composition paths.

Reply:
Resolved in 3818c6b